### ISSUE TITLE
White box with the dimensions of the first page, used as the background for all other pages when loading SWFs with [*,0]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#249]
```
What steps will reproduce the problem?

1. Create a PDF with Landscape orientation and convert to SWF: Test-1.swf
2. Create a second PDF with Portrait orientation and convert to SWF: Test-2.swf
3. Load them using: {Test-[*,0].swf,2}

Note, this does not occur if you create both pages in a single (2 page) PDF and 
convert it to a single SWF document loaded on it's own. In that case, the 
output displays as expected.

What is the expected output? What do you see instead?
Expect to see a landscape page as the first page and a portrait page as the 
second page, both pages displaying without any white spacing showing around 
them between the edge of the page and the background color in the PDF viewing 
area.

Instead, this is what is displayed: http://i.imgur.com/k5xmK.png

It appears as though FlexPaper assumes that every page is the same height / 
width as the first page when initially setting the container to display the 
page in BEFORE the page loads.

I've played around with this by changing the dimensions of the first page to 
all sorts of odd sizes, and exactly the same thing happens.

Also happens when a portrait page is first and a landscape page is second, but 
the effect is far less noticeable.


What version of the product are you using? On what operating system?
1.4.7 & 1.5.0. Error occuring on every operating system / browser tried 
(Windows, Mac, Ubuntu, IE, Firefox, Chrome).


Please provide any additional information below.

```

Original issue reported on code.google.com by `stevecom...@gmail.com` on 31 Dec 2011 at 4:14

**Comment by GoogleCodeExporter** [comment]
```
Is there any more information that I could provide that would help move this 
bug into "Accepted" status?
```

Original comment by `stevecom...@gmail.com` on 3 Feb 2012 at 6:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 3 Feb 2012 at 9:20
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Will this be fixed in the near future( 
http://groups.google.com/group/flexpaper/browse_thread/thread/ec6938d88beb8667 
)?
We did like to publish large books (500+ pages) that won't meet the 65536 
shapes SWF limit when compressed into a single file.
```

Original comment by `a.vannat...@gmail.com` on 17 May 2012 at 10:20

**Comment by GoogleCodeExporter** [comment]
```
fixed in 2.02 scheduled for release within a week
```

Original comment by `erik.eng...@devaldi.com` on 16 Oct 2012 at 8:03
- Changed state: **Fixed**
